doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,350 @@
+
+            Do not run the toolchain-manager installation as root (for example, using `sudo`).
+            It restricts access to the root user only, meaning you will need the root access for any subsequent installations.
+            If you run the script as root, to fix permissions delete the installation folder and run the script again as a non-root user.
+
+      #. Open a new terminal window with the |NCS| toolchain environment by running the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            /Users/*yourusername*/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "/Users/*yourusername*/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         .. note::
+            When working with west in the customer sampling release, you must always use a shell window with the |NCS| toolchain environment.
+
+      #. Run the following commands in your terminal to install the correct lxml dependency:
+
+         .. parsed-literal::
+            :class: highlight
+
+            pip uninstall -y lxml
+            pip install lxml
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+We recommend adding the nRF Util path to your environmental variables.
+
+Updating to the |NCS| 2.6.99-cs2
+********************************
+
+After you have updated the toolchain, complete the following steps to get the |NCS| v2.6.99-cs2:
+
+1. In the terminal window opened during the toolchain update, enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Updating the Terminal application
+*********************************
+
+To update `Serial Terminal from nRF Connect for Desktop`, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `Serial Terminal from nRF Connect for Desktop`.
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
+
+.. _migration_cs3_to_2_6_99_cs2_env_bringup:
+
+nRF54H20 DK bring-up
```

Comment:
Since this section is already there in the GSG, can't we include the content from there?

Reply:
I don't think so, as the migration guide will go on main and, in case, it will be updatable, while the gs guide will be static on the branch. So they might diverge in the future. Also, they are still slightly different, as one refers to updates and the other to installation. I'd like to keep them separate